boilerplate/app/app.tsx

```diff
@@ -73,6 +73,7 @@ function App(props: AppProps) {
 
   const [areFontsLoaded] = useFonts(customFontsToLoad)
 
+  // @mst replace-next-line React.useEffect(() => {
   const { rehydrated } = useInitialRootStore(() => {
     // This runs after the root store has been initialized and rehydrated.
```

Comment:
This comment is no longer true after removing MST, should probably also be removed.